composer.json

```diff
@@ -8,6 +8,7 @@
         "composer-plugin-api": "^1.0.0 || ^2.0.0",
         "aws/aws-sdk-php": "^3.257",
         "composer/composer": "^2.1.0",
+        "czproject/git-php": "^4.1",
```

Comment:
Why you don't use symfony/console? Process->run?

Reply:
From my perspective, the lib is helpful for the Git trigger purpose